CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## [0.25.2](https://github.com/nervosnetwork/neuron/compare/v0.25.1...v0.25.2) (2019-11-29)
+
+
+### Bug Fixes
+
+* remove bufferTime for address created event ([9b0a077](https://github.com/nervosnetwork/neuron/commit/9b0a077))
+* the missing txs ([ed557b6](https://github.com/nervosnetwork/neuron/commit/ed557b6))
+* **neuron-ui:** remove www from docs.nervos.org ([3fc8154](https://github.com/nervosnetwork/neuron/commit/3fc8154))
```

Comment:
```suggestion
* **neuron-ui:** remove www from docs.nervos.org ([3fc8154](https://github.com/nervosnetwork/neuron/commit/3fc8154))
```

It came with a new feature and I forgot to squash them in the PR.

Reply:
Not a problem I think.